explorer/e2e/testFunctions.ts

```diff
@@ -431,6 +443,18 @@ export async function testFilterCounts(
   return true;
 }
 
+/**
+ * Get a locator for a named filter tag
+ * @param page - a Playwright page object
+ * @param filterTagName - the name of the filter tag to search for
+ * @returns - a locator for the named filter tag
+ */
+const getFilterTagLocator = (page: Page, filterTagName: string): Locator => {
+  return page
+    .locator("#sidebar-positioner")
```

Comment:
Thoughts on adding a const for `#sidebar-positioner`?

Reply:
I was able to remove one use of it (it's not recommended by Playwright as it's implementation dependent), and I changed the other to a constant!